pkg/kubeclient/cpufrequency.go

```diff
@@ -25,7 +26,10 @@ import (
 const (
 	kubeturboNamespaceEnv = "KUBETURBO_NAMESPACE"
 	defaultNamespace      = "default"
-	defaultCpuFreq        = float64(2000) //MHz
+	defaultCpuFreq        = float64(2600) //MHz
```

Comment:
Are we changing the default to `2600` now?

Reply:
Sorry I missed your commit message, so I assume this must be agreed by @evat-pm ?

Reply:
Yes and she is ok with this change.